source/simulator/stokes_matrix_free.cc

```diff
@@ -938,7 +888,7 @@ namespace aspect
 
             pressure.evaluate (true,false,false);
             for (unsigned int q=0; q<pressure.n_q_points; ++q)
-              pressure.submit_value(cell_one_over_viscosity*pressure_scaling*pressure_scaling*
+              pressure.submit_value(one_over_cell_viscosity*pressure_scaling*pressure_scaling*
```

Comment:
would it make sense to move the ``*pressure_scaling*pressure_scaling`` into the ``VectorizedArray``?

Reply:
Yes, it would be cleaner.